File: programmers/문자열 정렬하기 (1).js
//** 숫자만 골라 오름차순 정렬한 리스트 */
function solution(my_string) {
  const answer = my_string.replace(/[^0-9]/g, "").split("");
  return answer.map((v) => Number(v)).sort((a, b) => a - b);
}

function solution(my_string) {
  return my_string
    .replace(/[^0-9]/g, "")
    .split("")
    .sort()
    .map((n) => Number(n));
}

File: programmers/python/문자열 정렬하기 (1).py
# 문자열 정렬하기 (1)
def solution(my_string):
    answer = []
    for i in my_string:
        if i.isdigit():
            answer.append(int(i))
    answer.sort()
    return answer


def solution(my_string):
    return sorted([int(c) for c in my_string if c.isdigit()])